Gruntfile.js

```diff
@@ -212,6 +255,38 @@ module.exports = function(grunt) {
 
 	});
 
+	grunt.registerTask( 'kebabToCamelCase', 'Rename any svg attributes to camel case for react', function() {
+		var svgFiles = grunt.file.expand( { filter: 'isFile', cwd: 'svg-min/' }, [ '**/*.svg' ] );
+
+		// Add stuff
+		svgFiles.forEach( function( svgFile ) {
+
+			// Grab the relevant bits from the file contents
+			var fileContent = grunt.file.read( 'svg-min/' + svgFile );
```

Comment:
Nothing really uses them after the build. I could make another tmp folder and delete both as a script if you think that looks cleaner.